src/utils/update/company.profile.mapper.to.overseas.entity.ts

```diff
@@ -1,18 +1,19 @@
 import { CompanyProfile } from "@companieshouse/api-sdk-node/dist/services/company-profile/types";
 import { yesNoResponse } from "../../model/data.types.model";
 import { Entity } from "../../model/entity.model";
-import { isSameAddress, mapAddress } from "../../utils/update/mapper.utils";
+import { isSameAddress, lowerCaseAllWordsExceptFirstLetters, mapAddress } from "../../utils/update/mapper.utils";
 
 export const mapCompanyProfileToOverseasEntity = (cp: CompanyProfile): Entity => {
   const serviceAddress = mapAddress(cp.serviceAddress);
   const principalAddress = mapAddress(cp.registeredOfficeAddress);
+
   return {
     registration_number: cp.foreignCompanyDetails?.registrationNumber,
     law_governed: cp.foreignCompanyDetails?.governedBy,
     legal_form: cp.foreignCompanyDetails?.legalForm,
-    incorporation_country: cp.jurisdiction,
+    incorporation_country: lowerCaseAllWordsExceptFirstLetters(cp.foreignCompanyDetails?.originatingRegistry?.country as string),
     public_register_name: cp.foreignCompanyDetails?.originatingRegistry?.name,
-    public_register_jurisdiction: cp.foreignCompanyDetails?.originatingRegistry?.country,
+    public_register_jurisdiction: "",
```

Comment:
undefined?

Reply:
is there a reason to do undefined instead of "" like we did with email? Was just following that.

Reply:
We should be consistent I agree. Safer to not set (or set to undefined) than inject some non null value like "" in general but probably ok to follow what we do for email.